.travis.yml

```diff
@@ -0,0 +1,171 @@
+# Copyright (c) 2013-2020 Arm Limited. All rights reserved.
```

Comment:
Please change the commit message to have a description of the targets added and why?

Also please don't include any issues in it, unless that is addressed in that commit which is fixing it. By default, issues addressed will get closed

Reply:
We previously had minimal commit messages and relied on PR description for details. The PR number can be retrieved from the merge commit. I'll elaborate the commit message.

Reply:
> https://github.com/ARMmbed/mbed-os-example-psa/pull/12#discussion_r540121071

But what if people ask why CYTFM_064B0S2_4343W is excluded? Is it kept track of?

Reply:
Updated commit message.

Reply:
Raised a issue - https://github.com/ARMmbed/mbed-os-example-psa/issues/14

Reply:
What I meant was, if we want a complete commit message, it should document known issues. I wasn't really asking that question.